collector/socket/src/main/cfg/org.apache.karaf.decanter.collector.socket.cfg

```diff
@@ -8,3 +8,6 @@
 
 # Protocol tcp(default) or udp
 #protocol=tcp
+
+# Dataformat json(default)
+#dataformat=json
```

Comment:
We should document (in the property comment and/or user guide) that `dataformat` property has to match the unmarshaller service property.

Reply:
Ok, I update the manual and add the comment in the property file

Reply:
Done in the documentation manual and the property file.